chia/wallet/wallet_nft_store.py

```diff
@@ -143,19 +175,66 @@ async def get_nft_by_id(self, nft_id: bytes32) -> Optional[NFTCoinInfo]:
             row[4] == IN_TRANSACTION_STATUS,
         )
 
-    async def rollback_to_block(self, height: int) -> None:
+    async def get_nft_by_coin_id(self, nft_coin_id: bytes32) -> Optional[NFTCoinInfo]:
```

Comment:
this function seems redundant. one could just call `get_nft_by_coin_ids()` instead and would only have to wrap the coin id in `[` and `]`.

If this function is used in a lot of places already, maybe it could be implemented in terms of `get_nft_by_coin_ids()`.